python-aexpect.spec

```diff
@@ -34,11 +34,12 @@ sftp, telnet, among others.
 
 %install
 %{__python} setup.py install --root %{buildroot} --skip-build
+mv %{buildroot}%{_bindir}/aexpect-helper %{buildroot}%{_bindir}/aexpect-helper-%{python_version}
```

Comment:
It looks like we need more than this... from https://fedoraproject.org/wiki/Packaging:Python#Naming :

"For example, the python3 version of "coverage" must ship executables /usr/bin/coverage-3 and /usr/bin/coverage-3.4 (assuming python3 is currently version 3.4), while the python2 version must provide /usr/bin/coverage, /usr/bin/coverage-2 and /usr/bin/coverage-2.7 (assuming python2 version 2.7)."

Reply:
I think this particular case falls in a grey area of the packaging guidelines.

As we already discussed, the only thing using the helper script is the library itself--which is now looking for it with the full -X.Y suffix. So, a version of the helper with only the -X suffix would be pointless. Packaging an un-suffixed version of the helper is also pointless, although the packaging guidelines say it must be provided and be for python2.

Of course, the library code could be further updated to also look for the helper with the -X suffix. But the build always creates the library artifacts with full X.Y versioning, so it doesn't seem proper to do that, either.

Would you still like me to add a symlink for the -X version?